Assets/Scripts/Game/Controllers/LootController.cs

```diff
@@ -16,6 +19,37 @@ public enum LootType { Module, RareMetal}
 		/// </summary>
 		public int RareMetals { get; private set; }
 
+		/// <summary>
+		/// Data for creating dropped module loot
+		/// </summary>
+		[SerializeField]
+		public LootData moduleDropData;
```

Comment:
Make this field private and then use a property to access this variable instead.

```
[SerializeField]
private LootData moduleDropData
public LootData ModuleDropData { //... determine if it needs to be get or set here }
```